pages/plugins/writing.md.erb

```diff
@@ -15,17 +17,17 @@ steps:
 ```
 {: codeblock-file="pipeline.yml"}
 
-## Create a new git repository
+### Create a new git repository
```

Comment:
I'm not sure we gain a lot by nesting everything as H3, other than moving it out of the in page TOC, which doesn't seem great.

Reply:
If it is to avoid clashing with the design topic, we could number the tutorial steps?

Reply:
I nested them because they're subsections of the tutorial. Can make them steps instead I guess (will work visually, not so nice from an HTML hierarchy pov)